src/orion/algo/ax/axoptimizer.py

```diff
@@ -133,8 +136,36 @@ def observe(self, trials):
            Trials from a `orion.algo.space.Space`.
 
         """
-        # TODO: Adapt this to your algo or remove if base implementation is fine.
-        super(AxOptimizer, self).observe(trials)
+        for trial in trials:
+            if not self.has_observed(trial):
+                # Check the trial status
+                trial_status = trial.status()
+
+                # If completed
+                if (trial_status == "completed"):
+                    # Get Ax trial
+                    with self.get_client() as client:
+                        # Complete it in Ax
+                        ax_trial_index = self._trials_map[trial.id] # tmp
+                        # ---- #
+                        params_in_orion_space = trial.params()
```

Comment:
```suggestion
                        params_in_orion_space = trial.params
```
`params` is also a property